lib/entice/web/client.ex

```diff
@@ -3,7 +3,7 @@ defmodule Entice.Web.Client do
   alias Entice.Web.Client
   alias Entice.Entity
   import Plug.Conn
-  import Entice.Web.Queries
+  alias Entice.Web.Queries
```

Comment:
For the sake of keeping the imports clean, i usually put this in the order of require, use, alias, import...
And within the same section I choose scope (internal deps first, external deps later on) and then by alphabet. So this should go between alias Entice.Web.Client and alias Entice.Entity (Entity is an external dep)